src/protocols/rdp/rdp.h

```diff
@@ -166,6 +181,17 @@ typedef struct guac_rdp_client {
      * reallocated during reconnection.
      */
     pthread_mutex_t rdp_lock;
+    
+    /**
+     * Condition which is used when the pthread needs to wait for a
+     * particular condition to be met.
```

Comment:
That particular condition needs to be pretty well defined, at least on our part, for waiting on this condition to make sense. If this condition and the associated flags are to be used for requesting arguments, for requesting credentials, etc. we can make this much clearer by specifically intending them for that use and documenting them as such.

Reply:
Okay, I've redefined this to be more specific to credential gathering.